### PR TITLE
PPLUS-331: Updated readme

### PR DESCRIPTION
**Pull request by VladislavStrelchenko**
Overview
Ticket: https://spryker.atlassian.net/browse/PPLUS-331

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,12 +9,45 @@ Upgrader is a tool that helps to update Spryker modules.
...
+```
+export GITHUB_ACCESS_TOKEN=<GITHUB_TOCKEN> # GitHub access token to project repository with permission to push branches and create PRs.
+export GITHUB_ORGANIZATION=<ORGANIZATION> # Free string.
+export GITHUB_REPOSITORY=<REPOSITORY> # Free string.
+```
+
+### Commands
+
+Upgrade command:
+1. Performs check that "is exists uncommitted changes in a project", if exist - the process will stop.
```
Comment (dereuromark) [inline README.md:37]:
```suggestion
1. Performs check that "there are no uncommitted changes in a project". If they exist, the process will stop.
```

File: README.md [hunk 2]
```diff
@@ -9,12 +9,45 @@ Upgrader is a tool that helps to update Spryker modules.
...
+2. Upgrades Spryker packages.
+3. Commits there changes in composer.json and composer lock files.
+4. Push changes and creates PR.
```
Comment (dereuromark) [inline README.md:40]:
```suggestion
4. Pushes changes and creates PR.
```
consistent usage of 3rd person language.

File: README.md [hunk 3]
```diff
@@ -9,12 +9,45 @@ Upgrader is a tool that helps to update Spryker modules.
...
+export GITHUB_ACCESS_TOKEN=<GITHUB_TOCKEN> # GitHub access token to project repository with permission to push branches and create PRs.
+export GITHUB_ORGANIZATION=<ORGANIZATION> # Free string.
+export GITHUB_REPOSITORY=<REPOSITORY> # Free string.
+```
+
+### Commands
+
+Upgrade command:
+1. Performs check that "is exists uncommitted changes in a project", if exist - the process will stop.
+2. Upgrades Spryker packages.
```
[the remark this answers is not in the document]
Comment (VladislavStrelchenko) [reply README.md]: Updated.

**Comment by VladislavStrelchenko** [comment]
@dereuromark thanks a lot for your help with these points.

**Pull request merged**